lib/ansible/module_utils/f5_utils.py

```diff
@@ -140,6 +140,10 @@ def fq_list_names(partition,list_names):
 
 # New style
 
+from abc import ABCMeta, abstractproperty
+from ansible.compat.six import with_metaclass
```

Comment:
IMHO `ansible.module_utils.six` should be used: `from ansible.module_utils.six import with_metaclass`